js/classes/entities/midground/chad.js

```diff
@@ -233,11 +226,14 @@ class Chad {
 
         // Dash action
         // When not CHAD is not on the ground and his dash is allowed and the user is trying to dash then enter conditional.
-        if (GAME.user.dashing && !this.isOnGround && this.canDash) {
+        this.dashTimer -= GAME.clockTick; // Decrease the jump timer
```

Comment:
Would you mind taking out this line too? I implemented functionality similar to this already

Reply:
Yah, assuming you mean the "this.canDash". You got it.

Reply:
I went ahead and removed all code relating to dashTimer that I implemented. Let me know if you need anything else. I'll be unavailable for the next few hours.